src/content-scripts/attention-collector.js

```diff
@@ -125,7 +128,7 @@
          * The title element contents of the page.
          * @type {string}
          */
-         let title = '';
+        let title = '';
```

Comment:
Same comment as above re: enforcing w/ eslint. I see it below too not gonna call out every one :) Let's just file a separate issue and `eslint --fix` it all at once separately.